runtime/src/main/java/org/corfudb/runtime/collections/CorfuTable.java

```diff
@@ -204,13 +83,13 @@ public int hashCode() {
 
     // The "main" map which contains the primary key-value mappings.
     private final ContextAwareMap<K,V> mainMap;
-    private final Set<Index<K, V, ? extends Comparable>> indexSpec;
+    private final Set<Index.Spec<K, V, ? extends Comparable>> indexSpec;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651443)